custom_components/foxess_modbus/entities/xx1_aux_entity_descriptions.py

```diff
@@ -22,6 +25,7 @@
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement="V",
         scale=0.1,
+        validate=[Range(0, 1000)],
```

Comment:
Ranges are exclusive, but 0 is a valid value. I'd be tempted to make all of the ranges inclusive